ajna/v2/modules/auctions.py

```diff
@@ -92,6 +92,12 @@ def _update_kick_locked_amount(chain, auction_uid, is_reward, locked_change):
 def process_kick_event(chain, event):
     borrower = event.data["borrower"].lower()
     bond = wad_to_decimal(event.data["bond"])
+    cwp = chain.current_wallet_position.objects.get(
```

Comment:
rename cwp to current_position and wp to position

Reply:
Also don't filter by `in_liquidation=False` as it's expected to be false on every kick event